src/transaction/boot_sr.c

```diff
@@ -5448,6 +5464,16 @@ xboot_emergency_patch (const char *db_name, bool recreate_log, DKNPAGES log_npag
       goto error_exit;
     }
 
+  {
+    const bool uses_remote_storage = prm_get_bool_value (PRM_ID_REMOTE_STORAGE);
+    if (uses_remote_storage)
+      {
+	er_set (ER_ERROR_SEVERITY, ARG_FILE_LINE, ER_TOOL_INVALID_WITH_REMOTE_STORAGE, 1, "'emergency_patchlog'");
```

Comment:
the indentation for this if block looks strange, is it git problem? If not I think it should be changed

Reply:
It's github's rendering of tabs at an arbitrary width.